sandbox/sandbox_darwin.go

```diff
@@ -0,0 +1,158 @@
+//go:build darwin
+
```

Comment:
@Mzack9999 suggesting a dummy/placeholder Sandbox implementation for platform/OS other than linux,darwin,windows .  I think we have seen some case in `utils` package where xyz user was having build failure since logic was not implemented for `netbsd`

Reply:
Done

Reply:
Good suggestion => Done